# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/Permissions.webidl
# Commit: 202c4f5c642b
# Full Hash: 202c4f5c642b8ec35e849c2144f935342d193fd7
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-05 21:29:44
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 3: Screen Wake Lock permission integration. r=webidl,dom-core,smaug,pbz,edgar
#   
#   Depends on D189509
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189510
# ==============================================================================

diff -r 82e03a404c2f -r 202c4f5c642b dom/webidl/Permissions.webidl
--- a/dom/webidl/Permissions.webidl	Tue Dec 05 15:37:05 2023 +0000
+++ b/dom/webidl/Permissions.webidl	Tue Dec 05 15:37:06 2023 +0000
@@ -13,7 +13,8 @@
   "push",
   "persistent-storage",
   "midi",
-  "storage-access" // Defined in https://privacycg.github.io/storage-access/#permissions-integration
+  "storage-access", // Defined in https://privacycg.github.io/storage-access/#permissions-integration
+  "screen-wake-lock" // Defined in https://w3c.github.io/screen-wake-lock/
 };
 
 [GenerateInit]